examples/nidcpower/measure-record.py

```diff
@@ -174,23 +174,24 @@ def on_close(event):
 
             plt.plot(x_axis, y_axis)
             plt.pause(0.001)
-
-            record_length += 50
-            x_axis = np.arange(start=1, stop=record_length+1, step=1)
             time.sleep(0.1)
             
     except KeyboardInterrupt:
         pass
 
     print(f"Effective measurement rate : {1/get_measure_record_delta_time.attribute_value}")
 
-    # close the session.
-    CheckForError(vi, (client.Close(nidcpower_types.CloseRequest(
-        vi = vi
-        ))).status)
-
 except grpc.RpcError as rpc_error:
     error_message = rpc_error.details()
     if rpc_error.code() == grpc.StatusCode.UNAVAILABLE :
         error_message = f"Failed to connect to server on {server_address}"
-    print(f"{error_message}") 
\ No newline at end of file
+    elif rpc_error.code() == grpc.StatusCode.UNIMPLEMENTED:
```

Comment:
As I mentioned in my other PR feedback, we should not be shipping a server to customers with unimplemented service handlers. Catching this from the examples communicates to the customer that there are some functions in the proto that just won't work.

Reply:
Whatever is finalized in the discussion on the PR of DMM example, I will update this example with same.
I have also made few other changes, Could you pl review it?